_docs/tasks/consul.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Running Istio in Docker with Consul
+overview: Make Istio run in docker environment by integrating Consul as a service registry.
+
+order: 42
```

Comment:
Do you have a screenshot showing how this appears ?

Reply:
I ran the following steps on my mac from the README:
```
$ xcode-select --install
$ brew install ruby
$ sudo gem install bundler
$ sudo gem install jekyll
$ cd istio.github.io
$ bundle install
$ bundle exec jekyll build
$ bundle exec jekyll serve
```
went to the Tasks page, made sure it was included in the list, and that the link went to my new page

Reply:
<img width="1091" alt="screen shot 2017-09-15 at 2 36 02 pm" src="https://user-images.githubusercontent.com/19473391/30500285-44bc284e-9a23-11e7-8921-0b72a6c9f674.png">

Reply:
<img width="1128" alt="screen shot 2017-09-15 at 2 37 12 pm" src="https://user-images.githubusercontent.com/19473391/30500325-68ea9566-9a23-11e7-884c-c97487882c93.png">

Reply:
seems like we should tweak the order? (priority setting)

Reply:
@ldemailly do you mean the order for my new item? or the order in general?

Reply:
Can you change the Installing Istio to Installing Istio in Kubernetes, and then change this to Installing Istio in Docker Swarm w/ Consul

Reply:
I also expect a follow up PR in the Integrating services into the mesh, where you need to have a special section for plain docker environments.